src/framework/components/light/component.js

```diff
@@ -49,12 +49,19 @@ var _lightPropsDefault = [];
  * entity.light.range = 20;
  * @property {string} type The type of light. Can be:
  * * "directional": A light that is infinitely far away and lights the entire scene from one direction.
- * * "point": A light that illuminates in all directions from a point.
- * * "spot": A light that illuminates in all directions from a point and is bounded by a cone.
+ * * "omni": A omni-directional light that illuminates in all directions from the light source.
+ * * "point": A omni-directional light but light source shape is {@link pc.LIGHTSHAPE_PUNCTUAL}.
```

Comment:
I think we are removing point light .. it's deprecated (from docs)

Reply:
I think we will need to keep `point` support for it backwards compatibility - but I will remove it from the jsdocs so that anyone reading the API will know to use `omni`

Reply:
done.